src/GraphList.hpp

```diff
@@ -50,6 +50,14 @@ template <typename VertexType, typename EdgeType> class GraphList {
       Exceptions::handle_exception_map(resp);
   }
 
+  // Helper method to call updateEdge method from PeakStore
+  template <typename E = EdgeType>
+  auto updateEdge(const VertexType &src, const VertexType &dest,
+               const EdgeType &newWeight)
+    -> std::enable_if_t<!CinderPeak::Traits::is_unweighted_v<E>, PeakStatus> {
```

Comment:
```suggestion
    -> std::enable_if_t<CinderPeak::Traits::is_weighted_v<E>, PeakStatus> {
```
Checking for negation would be the same i believe.

Reply:
Yeah